### PR TITLE
Added resolution to Project Roadmap connector missing in GUI of Data Policy

### PR DESCRIPTION
**Pull request by Markus-Hanisch**
…Policy in Power Platform admin center

[Project Roadmap connector is missing in data policy - data groups selection list](https://github.com/MicrosoftDocs/power-platform/issues/177#issuecomment-669308964)
The DLP policy connectors list shown in the GUI is missing the "Project Roadmap" ('shared_projectroadmap') connector. 

When you try to create a Project Roadmap and sync rows in the Project Roadmap with Microsoft Project projects (classic as well as modern), the auto-generated flow in Power Automate (see [Original template](https://emea.flow.microsoft.com/en-us/galleries/public/templates/0924e1395f0146599f7d174bc830cdc8/)) fails due to the following error message:
> Configured data policies prevent activation of flows which use the 'shared_commondataservice' and 'shared_projectroadmap' connectors. Visit the Power Automate Admin Center to view applied policies for your organization.

### CODE REVIEW

**Comment by Markus-Hanisch** [comment]
@efrene 
Could you review my commit? Have I made any mistake? Is anything missing?

**Comment by binaryjam** [comment]
I had something similar with another connector,
      Add-ConnectorToBusinessDataGroup -PolicyName $policyname  -ConnectorName shared_conversionservice

This worked,   but the same command with roadmap did not, but your version using add custom connector did.

However in the DLP UI they are listed as Custom Connectors, regardless of the mechnism I used here.   I suppose that will not matter because the full path ID is used,  I assume that is what is used as the Get DLP details cmdlet only shows that ID not the shortname.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: ProjectWeb/turn-project-for-the-web-off.md [hunk 1]
```diff
@@ -164,11 +164,36 @@ An admin can do the following to turn Roadmap on or off for their organization:
 1. In the Microsoft 365 admin center, expand the navigation menu, select **Settings**, and then select **Settings**.
 2. Select **Project**.
 3. On the **Project** settings page, select or deselect **Turn on Roadmap for your organization**, and then click **Save changes**.
-
-
  
 ![Roadmap Setting](media/projsettingsrm.png)
 
+4. Project Roadmap may requires the Project Roadmap connector to be added to a **[Data Policy](https://admin.powerplatform.microsoft.com/dlp)** in the Power Platform admin center.  At the moment, the [Project Roadmap connector does not show up in the GUI of the **Data Policy**](https://github.com/MicrosoftDocs/power-platform/issues/177).
```
Comment (JohanFreelancer9) [inline ProjectWeb/turn-project-for-the-web-off.md:170]:
```suggestion
4. Project Roadmap may require the Project Roadmap connector to be added to a [Data Policy](https://admin.powerplatform.microsoft.com/dlp) in the Power Platform admin center. At the moment, the [Project Roadmap connector does not show up in the GUI of the Data Policy](https://github.com/MicrosoftDocs/power-platform/issues/177). 
```

File: ProjectWeb/turn-project-for-the-web-off.md [hunk 2]
```diff
@@ -164,11 +164,36 @@ An admin can do the following to turn Roadmap on or off for their organization:
 1. In the Microsoft 365 admin center, expand the navigation menu, select **Settings**, and then select **Settings**.
 2. Select **Project**.
 3. On the **Project** settings page, select or deselect **Turn on Roadmap for your organization**, and then click **Save changes**.
-
-
  
 ![Roadmap Setting](media/projsettingsrm.png)
 
+4. Project Roadmap may requires the Project Roadmap connector to be added to a **[Data Policy](https://admin.powerplatform.microsoft.com/dlp)** in the Power Platform admin center.  At the moment, the [Project Roadmap connector does not show up in the GUI of the **Data Policy**](https://github.com/MicrosoftDocs/power-platform/issues/177). 
+Therefore, please follow along this guidance in order to add the Project Roadmap connector to the data group **Business** in the desired **Data Policy**.
```
Comment (JohanFreelancer9) [inline ProjectWeb/turn-project-for-the-web-off.md:171]:
```suggestion
Therefore, please follow this guidance in order to add the Project Roadmap connector to the data group **Business** in the desired **Data Policy**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by sayedpfe — CHANGES_REQUESTED** [review]
i believe your comment is valid and useful, but i think the location is not accurate, as you have added it to title "Turn Roadmap on or off", it should be a new title

**Comment by Markus-Hanisch** [comment]
@sayedpfe 
Well, to "turn on" the Project Roadmap feature, it is not enough to enable it via the Office 365 admin center.
I assume that the majority of all Office 365 tenants have set a DLP for Power Platform connectors.
And this DLP will block the Project Roadmap feature. Furthermore, the Project Roadmap connector cannot be added to a data group of the DLP DLP through the Power Platform admin center UI.
Therefore, in my opinion this PowerShell script is a necessary step to "turn on" the Project Roadmap feature, which is the reason why I decided to place the PR here.
If you have a more suitable location, kindly let me know :)

**Review by sayedpfe — APPROVED** [review]

**Comment by scanum** [comment]
@efrene PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by efrene — APPROVED** [review]

**Pull request merged**